src/Core/Application/IO/LoggingManager.hpp

```
@@ -160,10 +162,53 @@ class Log {
-		@param message: The message to be logged.
-		@param newline (default: true): A boolean determining whether the message ends with a newline character (true), or not (false).
+		@param messageFmt: The message to be logged, which can be formatted with curly braces (see std::format curly-braces-style string formatting).
+		@param args...: The message format arguments.
 	*/
-	static void Print(MsgType type, const char *caller, const std::string &message, bool newline = true);
+    template<typename... Args>
+	static void Print(MsgType type, const char *caller, const std::string_view messageFmt, Args&&... args) {
+		std::lock_guard<std::mutex> lock(m_printMutex);
+
+		// Get message type
+		std::string displayType = "Unknown message type";
```

Clang (14.0.6)

[warning] variable 'displayType' is not initialized

(cppcoreguidelines-init-variables)